libs/eo/metering-points/shell/src/lib/eo-metering-point-table.component.ts

```diff
@@ -115,14 +115,28 @@ import { EoBetaMessageComponent } from '@energinet-datahub/eo/shared/atomic-desi
         <mat-header-cell *matHeaderCellDef mat-sort-header>Granular Certificates</mat-header-cell>
         <mat-cell *matCellDef="let element">
```

Comment:
Suggestion: We should convert the table to the table from the design system, instead of using material directly.